<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1121</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QStackedWidget" name="stackedWidgetPrincipal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1141</width>
       <height>471</height>
      </rect>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>-11</x>
         <y>-11</y>
         <width>1131</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-10</y>
         <width>1131</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>-11</x>
         <y>-1</y>
         <width>1141</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>-11</x>
         <y>-11</y>
         <width>1141</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>1121</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-10</y>
         <width>1161</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_7"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProcesos"/>
   <addaction name="actionRendimiento"/>
   <addaction name="actionHistorial"/>
   <addaction name="actionInicio"/>
   <addaction name="actionUsuarios"/>
   <addaction name="actionDetalles"/>
   <addaction name="actionServicios"/>
  </widget>
  <action name="actionRendimiento">
   <property name="text">
    <string>Rendimiento</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionProcesos">
   <property name="text">
    <string>Procesos</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionInicio">
   <property name="text">
    <string>Inicio</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDetalles">
   <property name="text">
    <string>Detalles</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionServicios">
   <property name="text">
    <string>Servicios</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="text">
    <string>Usuarios</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionHistorial">
   <property name="text">
    <string>Historial</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
